composer.json

```diff
@@ -6,21 +6,35 @@
   "homepage": "https://5andhalf.com/",
   "minimum-stability": "dev",
   "prefer-stable": true,
-
   "require": {
-    "php": "^7.1.3",
+    "php": "^7.1",
     "laravel/framework": "^5.6",
```

Comment:
I'd like to update this, but we can't because of the laravel versions on the projects.